spec/views/users/users_index_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.describe 'User', type: :feature, js: false do
+  before do
+    # Create some sample users for testing
+    User.create(name: 'John', photo: 'https://example.com/john.jpg')
+    User.create(name: 'Alice', photo: 'https://example.com/alice.jpg')
+  end
+
+  it 'displays the list of users' do
+    visit '/users'
+
+    expect(page).to have_content('John')
+    expect(page).to have_content('Alice')
+  end
+
+  it 'links to user show pages' do
+    visit '/users'
+
+    click_link 'John'
+
+    user_id = User.find_by(name: 'John').id
+    expect(current_path).to eq("/users/#{user_id}")
+  end
+
+  it 'displays user information on show page' do
+    user = User.create(name: 'Jane', photo: 'https://example.com/jane.jpg')
+
+    visit "/users/#{user.id}"
+
+    expect(page).to have_content('Jane')
+    expect(page).to have_selector("img[src='https://example.com/jane.jpg']")
+  end
+end
```

Comment:
Good job making the unit tests. However, the following tests seem to not have been included:

- [ ] I can see the number of posts each user has written.

![image](https://github.com/HERMON-1995/Blog_app/assets/86318284/220b1f11-a13e-40fb-bec6-a1bb1fb12bb0)


- [ ] I can see the profile picture for each user.

![image](https://github.com/HERMON-1995/Blog_app/assets/86318284/25d7c5f6-6ab7-47d6-9d19-17eab4ebdfd4)